core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,61 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.io.File;
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
+
+    private String path;
+
+    /**
+     * Build object by setting file path to Path.toString().
+     * @param path file.
+     */
+    public FileEntry(Path path) {
+        this.path = path.toString();
```

Comment:
This creates a relative path, right? I think that absolute path is more appropriate. Imagine that we would collect several reports into one report - then we would need to count the real path according to the collected reports. I think that it would be also more useful in case of integration with ALM or Fabric8. 
WDYT?